README.md

```diff
@@ -7,11 +7,11 @@ MarkLogic Data Hub is a data integration platform and toolset that helps you qui
 
 # Version Support
 
-For Data Hub version 5.0, you need:
+For Data Hub version 5.2, you need:
 
-  - MarkLogic 9.0-10 or later
   - Gradle 4.6 or later
   - Java JDK 8 or later
+  - Check MarkLogic Version Compatibily [here](https://docs.marklogic.com/datahub/refs/version-compatibility.html)
```

Comment:
Looks like an older commit here - Compatibility is misspelled, and as mentioned before, I recommend making "Version Compatibility" a link.

Reply:
Yeah Looks like I cherry-picked a wrong commit. I'll pick the latest one. :)